apisix/plugins/openid-connect.lua

```diff
@@ -157,6 +157,10 @@ local schema = {
                 }
             },
         }
+        authorization_params = {
```

Comment:
@TrevorSmith-msr you need to add a `,` above this. Please fix the syntax error

Reply:
Thanks, I'll get that fixed.